canonical/ubuntu.com
# Implement /credentials/shop/manage as flask view

## Done
- Blocked by https://github.com/canonical/ubuntu.com/pull/13464
- Implementing `credentials/shop/manage` in Flask view, previously a React App
- Redesign tab selection to dropdown select
- Remove the option to rotate/redeem key if key has expired
- Update tooltip message when key is copied

## QA
- Go to https://ubuntu-com-13422.demos.haus/credentials/shop/manage
- Check that the dropdown tabs filter correctly
- Check that the keys are copied
- Check that the keys can be rotated repeatedly

## Issue / Card

Fixes [WD-7950](https://warthogs.atlassian.net/browse/WD-7950), [WD-9176](https://warthogs.atlassian.net/jira/software/c/projects/WD/boards/801?selectedIssue=WD-9176)


## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)


[WD-7950]: https://warthogs.atlassian.net/browse/WD-7950?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

[WD-9176]: https://warthogs.atlassian.net/browse/WD-9176?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
